### ISSUE TITLE
SubscribeToBbox Bug

### ISSUE THREAD

**Issue by david-ziegler** [kartevonmorgen/openfairdb#92]
subscribing to bbox fails. Error log: 
```
openfairdb::infrastructure::web::routes: Error: FOREIGN KEY constraint failed
```

only tested locally (v0.3.1)

**Comment by david-ziegler** [comment]
also on nightly.ofdb.io it doesn't work.
to reproduce: log in, click on "Kartenausschnitt abonnieren" --> response is "500 Internal Server Error"

**Comment by flosse** [comment]
should be fixed in fa301e06d78cabda0077466ce9366bbbd8ddb689
